app.vue

```diff
@@ -1,5 +1,10 @@
 <template>
   <div>
-    <NuxtPage />
+    <div>
+      <StdHeader />
+    </div>
+    <div>
+      <NuxtPage />
+    </div>
```

Comment:
Do these have to be a div or can they exist by themselves?

Reply:
They can exist by themselves in the main div, made the change.